mantidproject/mantid
# Muon Analysis results tab docs

**Description of work.**
Added documentation for results tab for muon analysis 2.
added sections for data selection window and fit function browser

**To test:**
Build documentation and check that documentations for Muon Analysis 2, its fitting tab and its results table tab are legible, suitably descriptive and that images load propperly.

Fixes #25767.
To be merged after pr #25805

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
